bin/config-yargs.js

```diff
@@ -7,6 +7,11 @@ const RESOLVE_GROUP = "Resolving options:";
 const OPTIMIZE_GROUP = "Optimizing options:";
 const INIT_GROUP = "Initialization:";
 
+const availableModes = () => {
+	const webpackOptionsSchema = require("webpack/schemas/WebpackOptions.json");
```

Comment:
Can we keep this statement on top of the file rather than doing inline require?

Reply:
If we move the `require` to the top of the file, perhaps with a less-redundant name like `optionsSchema`, why not just reference `optionsSchema.properties.mode.enum` directly in the `mode` object below? It would save the unnecessary indirection of creating a function that's only called in one place:

```js
const optionsSchema = require("webpack/schemas/WebpackOptions.json");

// ...

mode: {
	choices: optionsSchema.properties.mode.enum,
	// ...
}
```

Reply:
@bitpshr i'm actually doing that now. I refactored all that file to get all the informations from webpack schema. I will be updating it ASAP.